scene/resources/sky.cpp

```diff
@@ -82,7 +82,7 @@ void Sky::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_material", "material"), &Sky::set_material);
 	ClassDB::bind_method(D_METHOD("get_material"), &Sky::get_material);
 
-	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "sky_material", PROPERTY_HINT_RESOURCE_TYPE, "PanoramaSkyMaterial,ProceduralSkyMaterial,PhysicalSkyMaterial,ShaderMaterial"), "set_material", "get_material");
+	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "sky_material", PROPERTY_HINT_RESOURCE_TYPE, "PanoramaSkyMaterial,ProceduralSkyMaterial,PhysicalSkyMaterial,ShaderMaterial", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_EDITOR_INSTANTIATE_OBJECT), "set_material", "get_material");
```

Comment:
I'd argue procedural skies are about as common as using an environment texture, and users might not know both are even possible if it's instantiated by default. Maybe undo this particular change?

Reply:
I could reorder the hint so that a ProceduralSkyMaterial is created by default. In terms of usability, I'm not fond of the fact that you need to create a Sky resource *and* a SkyMaterial resource within for a sky to be visible – but at the same time, we can't automatically create a Sky in Environment because you won't need it if you're using a plain color background (which is typical for 2D projects).